scripts/generate-react-exports.js

```diff
@@ -52,13 +54,17 @@ for (const module of customElementsModules) {
   const source = `
   ${jsDoc}
   export const ${componentName}: React.LazyExoticComponent<ReactWebComponent<${componentType}>> =
-  customElements.whenDefined('${fileName}').then(() => createComponent<${componentType}>({
-    react: React,
-    displayName: "${componentName}",
-    tagName: "${fileName}",
-    elementClass: customElements.get("${fileName}"),
-    events: ${JSON.stringify(eventNames)},
-  }));
+  React.lazy(() =>
```

Comment:
How about using generics here instead of assigning a type to 'const'?

Reply:
How can we export the component in this way? we need to export a component in a synchronized way.

Reply:
We define the return type at line 56.

Reply:
Oh sorry, I misunderstood the question.